tracer/src/Datadog.Trace/SpanContext.cs

```diff
@@ -416,6 +415,14 @@ private static TraceId GetTraceId(ISpanContext context, TraceId fallback)
                    };
         }
 
+        /// <summary>
+        /// If <see cref="TraceContext"/> is not null, returns <see cref="Trace.TraceContext.GetOrMakeSamplingDecision"/>.
+        /// Otherwise, returns <see cref="SamplingPriority"/>.
+        /// </summary>
+        internal int? GetOrMakeSamplingDecision() =>
```

Comment:
> with our current trace model

big `SpanContext` refactoring coming soon? haha

Reply:
Hmm, maybe we can default to `1` like we're now doing in `TraceContext.GetOrMakeSamplingDecision()`. I'll explore this, but I may have to do this in a separate PR because Dash.